operator/controllers/csiscaleoperator_controller.go

```diff
@@ -242,41 +234,29 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		}
 
 		if err := r.deleteClusterRolesAndBindings(instance); err != nil {
-			message := "Failed to delete ClusterRoles and ClusterRolesBindings"
+			message := "Failed to delete the ClusterRoles and ClusterRoleBindings"
```

Comment:
Here it would be better if we tell what all clusterrole and rolebindings are stuck or we should give the hint to check those such as 

Failed to delete the ClusterRoles and ClusterRoleBindings. To get the list use selector field as "--selector='product=ibm-spectrum-scale-csi' "